dialect/sql/schema/postgres.go

```diff
@@ -229,13 +229,14 @@ const maxCharSize = 10 << 20
 // scanColumn scans the information a column from column description.
 func (d *Postgres) scanColumn(c *Column, rows *sql.Rows) error {
 	var (
-		nullable         sql.NullString
-		defaults         sql.NullString
-		udt              sql.NullString
-		numericPrecision sql.NullInt64
-		numericScale     sql.NullInt64
+		nullable                      sql.NullString
+		defaults                      sql.NullString
+		udt                           sql.NullString
+		numericPrecision              sql.NullInt64
+		numericScale                  sql.NullInt64
+		numericCharacterMaximumLength sql.NullInt64
```

Comment:
```suggestion
		characterMaximumLen sql.NullInt64
```

The `numeric` prefix is not part of the column name.

Reply:
done